plugins/check_fping.c

```diff
@@ -60,6 +60,8 @@ static int packet_count = PACKET_COUNT;
 static int target_timeout = 0;
 static int packet_interval = 0;
 static bool verbose = false;
+static bool dontfrag = false;
+static bool random = false;
```

Comment:
Please rename that symbol to avoid a conflict with `random(3)`. I would suggest `randomize_packet_data`

Reply:
No problem, I've just pushed the update to the PR :)